tools/codegen/gen_backend_stubs.py

```diff
@@ -205,6 +205,7 @@ def make_file_manager(install_dir: str) -> FileManager:
                 'extra_cuda_headers': '',
                 'legacy_th_headers': '',
                 'external_backend_headers': external_backend_headers,
+                'namespaced_headers': '',
```

Comment:
I haven't actually started codegen'ing the `XLAFunctions.h` file that exposes the `at::xla::{op}` fast path API - when we eventually do that we'll need the same logic here.